<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Книга</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="12" column="1">
           <widget class="QCheckBox" name="cB_Statia">
            <property name="text">
             <string>Статья из сборника</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QLineEdit" name="LE_pages"/>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Год</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Название издательства</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Стр</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Доп строка (если есть)</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QCheckBox" name="cB_Elec">
            <property name="text">
             <string>Электронный ресурс</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="CB_author"/>
          </item>
          <item row="11" column="0">
           <widget class="QLineEdit" name="LE_Year"/>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>CheckBox</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLineEdit" name="LE_Dop"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Индекс</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="LE_Name"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Наименование</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QComboBox" name="CB_Index"/>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="But_Add">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLineEdit" name="LE_Izd"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Авторы</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Страница</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>751</width>
          <height>281</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="4" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Дата обращения</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Название с заглавия</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>URL</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="LE_siteName"/>
         </item>
         <item row="6" column="0">
          <widget class="QLineEdit" name="LE_SiteDate"/>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="LE_siteURL"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Статья</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="6" column="1">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Стр</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>Номер выпуска</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Журнал издатель</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>Авторы</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="LE_pages_4"/>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="LE_Name_4"/>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="CB_author_4"/>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="LE_num"/>
          </item>
          <item row="7" column="0">
           <widget class="QLineEdit" name="LE_Year_4"/>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="But_Add_4">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Год</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLineEdit" name="LE_Izd_4"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Наименование</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="But_Show">
      <property name="text">
       <string>Вывести</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="But_Save">
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
